ansible/playbook.yml

```diff
@@ -6,51 +6,34 @@
     email: "lucas@dohmen.io"
     url: "https://lucas.dohmen.io"
 
-    go_version: 1.9
-    nodejs_version: 8.x
-    ruby_version: 2.4
-    compose_version: 1.17.0-rc1
-    compose_checksum: sha1:27f3648e33ee5c8f68c1d05c9e51a5b6669f94bd
+    nodejs_version: 10.x
+    ruby_version: 2.5
 
     apt_keys:
-      A14F4FCA: 'http://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc'
       68576280: 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
       0EBFCD88: 'https://download.docker.com/linux/ubuntu/gpg'
     apt_repositories:
-      - 'deb https://packages.erlang-solutions.com/ubuntu xenial contrib'
-      - 'deb https://deb.nodesource.com/node_{{ nodejs_version }} xenial main'
-      - 'deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable'
+      - 'deb https://deb.nodesource.com/node_{{ nodejs_version }} bionic main'
+      # TODO: Replace with bionic when it is available
```

Comment:
This is some weird shit... The repo exists, but it doesn't contain docker-ce...

Reply:
It is empty right now, I am using the repo for the previous debian version til this is fixed.

Reply:
Yep, that's what I do as well 😄